salt/utils/parsers.py

```diff
@@ -3370,6 +3371,11 @@ def _mixin_setup(self):
                 "into the default roster file (flat)."
             ),
         )
+        auth_group.add_option(
+            "--pki-dir",
+            dest="pki_dir",
+            help=("Set the directory to load the pki keys."),
```

Comment:
```suggestion
            help="Set the directory to load the pki keys.",
```

Reply:
Sure, I can make that change when I revise the PR.